Web/Controller/LandingPages.hs

```diff
@@ -38,6 +38,38 @@ instance Controller LandingPagesController where
                     render EditView { .. }
                 Right landingPage -> do
                     landingPage <- landingPage |> updateRecord
+
+                    -- After we update the Landing page, we can set the order of the paragraphs.
+                    let params = paramListOrNothing @UUID "paragraphId"
+
+                    case catMaybes params of
+                        [] -> do
+                            -- No paragraphs to update.
+                            pure ()
+
+                        uuids -> do
+                            -- We need to update the weight of the paragraphs,
+                            -- So load them.
+                            landingPage <- fetchLandingPageWithParagraphs landingPageId
+
+                            -- Iterate over all paragraphs, and update the weight.
+                            forEach landingPage.paragraphCtas updateParagraph
+
+                            -- @todo: If I uncomment this, it results with a compiler error.
+                            -- forEach landingPage.paragraphQuotes updateParagraph
```

Comment:
The error we get

![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/b324b88b-752d-4b22-a87c-650539b59506)